lib/rubocop/cop/lint/void.rb

```diff
@@ -26,6 +26,15 @@ module Lint
       #
       # @example
       #
+      #   # bad, when NonmutatingMethods is set true
+      #
+      #   def some_method(some_array)
+      #     some_array.sort
+      #     do_something
```

Comment:
I'd have `do_somethin` operate on `some_array`, otherwise this example doesn't make much sense.